bin/dartdoc.dart

```diff
@@ -152,21 +152,21 @@ main(List<String> arguments) async {
     generator.onFileCreated.listen(_onProgress);
   }
 
-  var addCrossdart = args['add-crossdart'] as bool;
-  var includeSource = args['include-source'] as bool;
-
   DartSdk sdk = new FolderBasedDartSdk(PhysicalResourceProvider.INSTANCE,
       PhysicalResourceProvider.INSTANCE.getFolder(sdkDir.path));
 
   setConfig(
-      addCrossdart: addCrossdart,
+      addCrossdart: args['add-crossdart'] as bool,
```

Comment:
I'm curious if the analyzer needs this casting - do you get a warning without the `as bool`?